_includes/nav/page-item.html

```diff
@@ -1,8 +1,8 @@
 {% include nav/current.html page=include.page %}
 <a href="{{ include.page.url | relative_url }}"{%
-  if _current -%} class="usa-current"{% endif %}>{{
-    include.page.title | smartify
```

Comment:
_Re-adding in this new PR_ 
@donjo do you know how I can add the `<span>` to the header nav but not the sidenav?